### PR TITLE
Triton Server CV TensorFlow model

### PR DESCRIPTION
**Pull request by ashrawat**
*Issue #, if available:*

*Description of changes:* more Black-nb suggestions

*Testing done:* Yes

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x] I have tested my notebook(s) and ensured it runs end-to-end
- [x] I have linted my notebook(s) and code using `black-nb -l 100 {path}/{notebook-name}.ipynb`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: inference/cv/realtime/Triton/multi-model/tensorflow-backend/triton-cv-mme-tensorflow-backend.ipynb [hunk 1]
```diff
@@ -0,0 +1,1532 @@
+{
```
Comment (atqy) [inline inference/cv/realtime/Triton/multi-model/tensorflow-backend/triton-cv-mme-tensorflow-backend.ipynb:1]: <p>Can you clear all outputs of the notebook cells? It makes the notebooks cleaner</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3847/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (atqy) [inline inference/cv/realtime/Triton/multi-model/tensorflow-backend/triton-cv-mme-tensorflow-backend.ipynb:1]: <p>Get rid of the empty line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3847/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Comment (atqy) [inline inference/cv/realtime/Triton/multi-model/tensorflow-backend/triton-cv-mme-tensorflow-backend.ipynb:1]: <p>Ah, this bucket is no longer maintained. Please use s3://sagemaker-sample-files/datasets/image/MNIST instead</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3847/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

File: inference/cv/realtime/Triton/single-model/tensorflow-backend/triton-cv-sme-tensorflow-backend.ipynb [hunk 2]
```diff
@@ -0,0 +1,1371 @@
+{
```
Comment (atqy) [inline inference/cv/realtime/Triton/single-model/tensorflow-backend/triton-cv-sme-tensorflow-backend.ipynb:1]: <p>Can you clear all notebook cell outputs? It makes the notebook cleaner</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3847/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (atqy) [inline inference/cv/realtime/Triton/single-model/tensorflow-backend/triton-cv-sme-tensorflow-backend.ipynb:1]: <p>Ah, this bucket is no longer maintained. Please use s3://sagemaker-sample-files/datasets/image/MNIST instead</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3847/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (atqy) [inline inference/cv/realtime/Triton/single-model/tensorflow-backend/triton-cv-sme-tensorflow-backend.ipynb:1]: <p>Please clear the blank line in this cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3847/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

**Comment by ashrawat** [comment]
The feedback is implemented in the next pull request.

**Pull request closed without merging**